### PR TITLE
Renamed folder, added subsection summaries

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -104,7 +107,7 @@ Nighttime Operations
 Troubleshooting
 ======================
 
-All related to troubleshooting is included in this section. 
+This section features information about how to recover from faults during afternoon checkout or nighttime operations. If the problem persists after exhausting all documented procedures, ensure safety of the equipment, stop work, and call for help.
```
Comment (patrickingraham) [inline index.rst:110]: So this section should be about unexpected faults that aren't covered elsewhere in the documentation.
Consider:
`This section includes information on current known issues that are not addressed in other areas of the documentation.
Content includes temporary issues and potential work-arounds. 
The section is continually updated as the observatory system evolves and develops.`

**Comment by Slimleashma** [comment]
All of Patrick's comments were addressed.

File: index.rst [hunk 2]
```diff
@@ -39,10 +39,12 @@ All aspects of the project that go through strict change control (e.g. mirror pr
 
 .. _Obs-Ops-Safety:
 
-Safety
-======
+Quick Safety Reference for Nighttime Personnel
```
Comment (tribeiro) [inline index.rst:42]: I think having "Safety" as the first word in the title would be better.. Maybe something like:


```
Safety: Quick Reference for Nighttime Personnel
```

or, preferably, in my opinion:

```
Safety: Quick Reference Guidelines
```

File: Operational-Tutorials/index.rst [hunk 3]
```diff
@@ -14,16 +14,16 @@
...
 .. Each section should includes a label for cross referencing to a given area.
 .. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
 .. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
 .. An error will alert you of identical labels during the build process.
 
-###############################################################
-Operational Procedures - System testing, deployment and upgrade 
-###############################################################
+#########################################
+System testing, deployments, and upgrades
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Operational-Tutorials/index.rst:25]: Why was this marked as resolved?
Just an FYI that the person who writes the comment is the one who marks it as resolved.
To show that you've implemented the change without needing to write anything, the practice is to add the 👍 emoji to the comment. See the following link:
https://developer.lsst.io/work/flow.html#code-review-discussion

**Review by tribeiro — APPROVED** [review]
Looks good to me..

File: Operational-Tutorials/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (Slimleashma) [reply Operational-Tutorials/index.rst:25]: 👍

File: index.rst [hunk 4]
```diff
@@ -42,7 +42,7 @@ All aspects of the project that go through strict change control (e.g. mirror pr
 Safety
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst [hunk 5]
```diff
@@ -42,7 +42,7 @@ All aspects of the project that go through strict change control (e.g. mirror pr
...
 ======
 
-Content in this section relates to personnel safety and resources.
+Safety of personnel first and equipment second is the utmost priority of Rubin Observatory. The safety information here is meant to serve as a quick look for nighttime support staff in case of an emergency on the summit. It does not replace the required safety training to be on the summit at night.
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst [hunk 6]
```diff
@@ -64,6 +64,7 @@ This includes:
 Daytime Operations
 ======================
 
+This section encapsulates procedures for normal operations, such as calibrations routines and standard telescope checkout. Any non-standard procedures, such as shutdown recovery and non-standard operations are not included here. 
 .. toctree::
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst:74]: 👍

File: index.rst [hunk 7]
```diff
@@ -92,6 +93,8 @@ This area contains information related to the summit operations and the handoff
 Nighttime Operations
 ======================
 
+Standard nighttime operations are listed in this section. This includes opening the telescopes for data collection and end-of-night procedures. Special operations or engineering tasks are described elsewhere.
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst [hunk 8]
```diff
@@ -144,9 +147,9 @@ Contains detailed instructions on how to set the required environment for operat
     Observing-Interface-Setup/index
 
 
-.. _Obs-Ops-Operational-Procedures-Tutorials:
+.. _Obs-Ops-Operational-Tutorials:
 
-Operational Procedures - System testing, deployment and upgrade
+System testing, deployments, and upgrades
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 1 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst [hunk 9]
```diff
@@ -39,10 +39,12 @@ All aspects of the project that go through strict change control (e.g. mirror pr
...
 .. _Obs-Ops-Safety:
 
-Safety
-======
+Quick Safety Reference for Nighttime Personnel
+==============================================
 
-Safety of personnel first and equipment second is the utmost priority of Rubin Observatory. The safety information here is meant to serve as a quick look for nighttime support staff in case of an emergency on the summit. It does not replace the required safety training to be on the summit at night. 
+Safety of personnel first and equipment second is the utmost priority of Rubin Observatory. 
+The safety information here is meant to serve as a quick look for nighttime support staff in case of an emergency on the summit.
```
[the remark this answers is not in the document]
Comment (Slimleashma) [reply index.rst]: 👍

**Comment by Slimleashma** [comment]
Merged

**Pull request merged**